scripts/pin_the_slow_way.py

```diff
@@ -25,33 +25,80 @@
 import conda_smithy.feedstocks as feedstocks
 
 
-pinned = {
+# Some links that point to backward compatibility
+# https://abi-laboratory.pro/tracker/timeline/ffmpeg/
+# https://abi-laboratory.pro/tracker/timeline/libtiff/
+# https://abi-laboratory.pro/tracker/timeline/libzip/
+# https://abi-laboratory.pro/tracker/timeline/fontconfig/
+# https://abi-laboratory.pro/tracker/timeline/freetype/
+# https://abi-laboratory.pro/tracker/timeline/openssl/
+# https://abi-laboratory.pro/tracker/timeline/sqlite/
+# http://upstream.rosalinux.ru/versions/libgeos_c.html
+# http://upstream.rosalinux.ru/versions/libgeos.html
+# http://upstream.rosalinux.ru/versions/netcdf.html
+# http://upstream.rosalinux.ru/versions/readline.html
+# http://upstream.rosalinux.ru/versions/tk.html
+# http://upstream.rosalinux.ru/versions/libsox.html
+# http://upstream.rosalinux.ru/versions/zlib.html
+# http://upstream.rosalinux.ru/versions/ncurses.html
+# http://upstream.rosalinux.ru/versions/blitz.html
+
+build_pin = {
           'boost': 'boost 1.61.*',
           'bzip2': 'bzip2 1.0.*',
           'ffmpeg': 'ffmpeg 2.8.*',
           'fontconfig': 'fontconfig 2.11.*',
           'freetype': 'freetype 2.6.*',
           'geos': 'geos 3.4.*',
-          'giflib': 'giflib 5.1.*',
+          'giflib': 'giflib 5.1.2',
```

Comment:
That is exactly my understating. BTW that one new symbol may fail when tying to use that particular `symbol`. In general things would work in your scenario.